src/cook-web/src/app/c/[[...id]]/Components/CourseCatalog/CourseCatalog.module.css

```
@@ -0,0 +1,53 @@
+  height: 0;
+  overflow: hidden;
+}
+.courseCatalog .titleRow {
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  justify-content: space-between;
+  cursor: pointer;
+  height: 24px;
+  overflow: hidden;
+  overflow: hidden;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

overflow is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)